### PR TITLE
506 request hosting info panel

### PR DESCRIPTION
**Pull request by vssdeo**
To complement #422 Quickreplies / hosting requests, when the first person is sending the first message, he can:

*while requesting to be hosted, click a checkbox & give details of
- number of people
- date from
- date to

### CODE REVIEW

File: modules/messages/client/controllers/thread.client.controller.js [hunk 1]
```diff
@@ -32,6 +32,12 @@
         messageIdsInView = [],
         editorContentChangedTimeout;
 
+    // Vars for Host Module
+    var fromDateDisp,
```
Comment (mrkvon) [inline modules/messages/client/controllers/thread.client.controller.js]: It would be nice to have short comments about purpose of the variables here.

Comment (vssdeo) [inline modules/messages/client/controllers/thread.client.controller.js]: Ah ! Sure I was putting comments for functions need to put more here, I think I will put more in detail in thread dimension directive & other little places too !

**Comment by vssdeo** [comment]
@mrkvon added the comments explaining variables

**Comment by simison** [comment]
We might get back to implementing this at some point but for now, it's not on our agenda.

**Pull request closed without merging**